sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample12_ConsumeEventsWithEventProcessor.cs

```diff
@@ -78,7 +67,76 @@ await using (var client = new EventHubClient(connectionString, eventHubName))
 
                 // Let's finally create our event processor.  We're using the default consumer group that was created with the Event Hub.
 
-                var eventProcessor = new EventProcessor<SamplePartitionProcessor>(EventHubConsumer.DefaultConsumerGroupName, client, partitionManager, eventProcessorOptions);
+                var eventProcessor = new EventProcessor(EventHubConsumer.DefaultConsumerGroupName, client, partitionManager, eventProcessorOptions);
+
+                int totalEventsCount = 0;
+                int partitionsBeingProcessedCount = 0;
+
+                // TODO: explain callbacks setup once the public API is finished for the next preview.
```

Comment:
Do we need an issue to track revisiting this?

Reply:
Not really. I intend to get this done for the fifth preview.